spec/features/blacklight_customizations/zero_results_spec.rb

```diff
@@ -20,23 +20,26 @@
     end
   end
 
-  describe 'sidebar' do
-    context 'when a user can access the chat widget' do
-      let(:user) { User.new(email: 'example@stanford.edu') }
-
-      scenario 'the widget is rendered' do
+  describe 'search context sidebar' do
+    context 'catalog' do
+      scenario 'links to articles+' do
         within '#sidebar' do
-          expect(page).to have_css('h3', text: 'Want help?')
-          expect(page).to have_css('h3', text: 'On the library website')
-          expect(page).to have_css('a', count: 4)
+          expect(page).to have_css('a', text: 'articles+')
+          expect(page).to have_css('a', text: 'library website')
+          expect(page).to have_css('a', text: 'all')
+          expect(page).to have_css('a', count: 3)
         end
       end
     end
 
-    context 'when a user cannot access the chat widget' do
-      scenario 'the widget is not rendered' do
+    context 'articles' do
+      scenario 'articles+' do
```

Comment:
Can you add a better description of the scenario?

Reply:
✅